desktop/src/hooks/useFolderList/useFolderList.ts

```diff
@@ -0,0 +1,148 @@
+import { fetchProtectedRoute } from '@/preload/api/sso';
```

Comment:
I know this works but the idea of the preload is that we create functions there that are accessible to the renderer via window.api because the renderer does not have access to nodejs modules or lower level programming from the main script.

Reply:
Its probably better if you got this from window.api just like you do on line 10